builds/any/rootfs/stretch/common/arm64-base-packages.yml

```diff
@@ -13,3 +13,4 @@
 - dnsutils
 - mft
 - mrvl-fw-image
+- keepalived
```

Comment:
Why not add it for all architectures?

Reply:
I want to use a package from dent artifacts https://github.com/dentproject/dent-artifacts/blob/master/REPO/stretch/packages/binary-arm64/keepalived_2.0.10-1_arm64.deb

Reply:
Please mention that in the commit message. Also, is it documented somewhere, why version 2.1.5-0.2 from the official Debian repository (upstream version is 2.2) can’t be used?

Reply:
The latest version for strech is 1.3.2.1 https://packages.debian.org/stretch/keepalived. 
2.0.10-1 is the version used in buster https://packages.debian.org/source/stable/keepalived.
the reason for updating the package is bugs fixed in this release. for example this one *https://github.com/acassen/keepalived/issues/905**

> Please mention that in the commit message.
I've updated the commit message and pull request description to include info about where the package is taken from

Reply:
Thank you for the update. Maybe instead of

> Use arm64 package form dent-atrifacts[3]

use

> Use arm64 package from [dent-artifacts][3], as the version 1.3.2-1 in Debian 9 (stretch) is too old. Also, dent-artifacts only contains the build for arm64 and not amd64.

Reply:
sure. will update commit message

Reply:
> Use the arm64 package from [dent-artifacts][3], as the version 1.3.2-1 in Debian 9 (stretch) is too old. Also, dent-artifacts only contains the build for arm64 and not amd64.

done